ext/com_dotnet/com_wrapper.c

```diff
@@ -257,13 +257,10 @@ static HRESULT STDMETHODCALLTYPE disp_invokeex(
 		/* TODO: if PHP raises an exception here, we should catch it
 		 * and expose it as a COM exception */
 
-		if (wFlags & DISPATCH_PROPERTYGET) {
-			retval = zend_read_property(Z_OBJCE(disp->object), Z_OBJ(disp->object), Z_STRVAL_P(name), Z_STRLEN_P(name), 1, &rv);
-			ret = S_OK;
-		} else if (wFlags & DISPATCH_PROPERTYPUT) {
+		if (wFlags & DISPATCH_PROPERTYPUT) {
 			zend_update_property(Z_OBJCE(disp->object), Z_OBJ(disp->object), Z_STRVAL_P(name), Z_STRLEN_P(name), &params[0]);
 			ret = S_OK;
-		} else if (wFlags & DISPATCH_METHOD) {
+		} else if (wFlags & DISPATCH_METHOD && zend_is_callable_ex(name, Z_OBJ(disp->object), 0, NULL, NULL, NULL)) {
```

Comment:
What may make sense as a follow-up is to pass an FCC to get rid of the `call_user_function()` call below as we would be able to just call it, instead of rechecking it.

Nor do I think the `zend_try { } zend_catch { }` is necessary (except if it needs to deal with Fatal Errors? But that doesn't seem to be what the comment is implying).

Reply:
Regarding the FCC refactoring, see #17001.

Regarding the try-catch: this is not related to the comment above (which still needs to be addressed), but it's likely necessary for out-of-process COM invocation, since even if the ZE bails out, the caller needs to be informed. However, the current implementation might not necessarily work. Might need to move the try-catch up the call stack, and retrigger `zend_bailout()` ASAP.

Reply:
Thank you for working on this, it is highly appreciated!